encoding/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package encoding provides utilities for encoding values for specific
+// document encodings.
+
+package encoding
```

Comment:
Nit: At a later point, we can consider updating this page to have link to available encoding packages